### ISSUE TITLE
RuntimeError: CorrMM failed to allocate working memory of 1 x 1600 x 640000

### ISSUE THREAD

**Issue by sitrucp** [alexjc/neural-enhance#6]
I am getting error like below. Is this as simple as my system memory is being exhausted?

I am using it on Windows 64, 8GB, i7 4500 CPU @ 1.8GHz

```
Traceback (most recent call last):
  File "C:\lasagne\WinPython\python-3.4.4.amd64\lib\site-packages\theano\compile\function_module.py", line 866, in __call__
    self.fn() if output_subset is None else\
RuntimeError: CorrMM failed to allocate working memory of 1 x 576 x 2849600
```

**Comment by alexjc** [comment]
What size of image are you inputting? There's currently no tiling of images in the code, I will think about how best to add it.

Try `--model=small` to reduce memory footprint to.

**Comment by sitrucp** [comment]
Hey thanks

 

File is 200x200 px 32 bit png

 

Same thing with –model-small (using https://github.com/alexjc/neural-enhance/releases/download/v0.1/ne4x-small-0.1.pkl.bz)

**Comment by hammertimedk** [comment]
I had to increase my swapfile to about 32 GB to be able to handle a 500x600px image - which on CPU setting then took ages to process.

Also I had very little success with the test images I tried. I just ended with scaled up versions of the same image with very little noticeable difference. I'll be trying more samples before I start complaining and opening an issue, but for now I'm not seeing the CSI-style enhancement I was hoping to get :D
